livefs_edit/actions.py

```diff
@@ -232,6 +232,13 @@ def inject_snap(ctxt, snap, channel="stable"):
     rootfs = setup_rootfs(ctxt)
     seed_dir = f'{rootfs}/var/lib/snapd/seed'
     snap_mount = ctxt.add_mount('squashfs', snap, ctxt.tmpdir())
+
+    snap_preseed_binary = '/snap/core/current/usr/lib/snapd/snap-preseed'
```

Comment:
I think using the one from the 'snapd' snap would be more future proof?

Reply:
Updated and changed it to `/snap/snapd/...`

Reply:
thanks